components/02-molecules/menus/main-menu/_02-main-menu-link.scss

```diff
@@ -17,7 +17,7 @@
     color: $gray-dark;
     display: inline-block;
     font-size: 1.1rem;
-    padding: $space $space-double $space $space;
+    padding: $space $space $space $space;
```

Comment:
Let's just change this to `padding: $space;`

Reply:
done